src/makeCalculator.js

```diff
@@ -4,6 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      this.result /= number;
```

Comment:
The `divide` method should handle division by zero to prevent potential errors. Consider adding a check to ensure `number` is not zero before performing the division.